<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QT-LED控制程序</string>
  </property>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>执行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>150</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>LED1控制</string>
   </property>
   <widget class="QRadioButton" name="radioButton" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>打开</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>关闭</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>LED2控制</string>
   </property>
   <widget class="QRadioButton" name="radioButton_3" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>打开</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>关闭</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>160</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>LED3控制</string>
   </property>
   <widget class="QRadioButton" name="radioButton_5" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>打开</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>关闭</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
